src/socketTx.ts

```diff
@@ -60,15 +60,40 @@ export class SocketTx {
     return allowanceValue.lt(minimumApprovalAmount);
   }
 
-  private _validateSend(send: {
+  private async _validateSend(send: {
     data?: string | undefined;
     to?: string | undefined;
     from?: string | undefined;
   }) {
     if (this.userTxType === UserTxType.FUND_MOVR) {
-      const addresses = Object.values(socketAddresses[this.chainId]);
-      if (!addresses.includes(send.to)) {
-        throw new Error(`${send.to} is not a recognised socket address on chain ${this.chainId}`);
+      // Fetcg route.
+      const routeResponse = await Routes.getActiveRoute({ activeRouteId: this.activeRouteId });
+      if (!routeResponse.success) throw new Error("Error while fetching route.");
+      const route = routeResponse.result;
+
+      // Current User Tx - Bridge Name
+      const currentUserTx = route.userTxs[this.userTxIndex];
+      const bridgeStep = currentUserTx.steps.find((step) => step.type === 'bridge');
```

Comment:
It's a shame that the `nextTxResponse` does not include the steps, it would avoid the extra call

Reply:
Yeah. We decided to keep it clean. 
Maybe in the next version.